source/cursorManager.py

```diff
@@ -29,13 +29,19 @@
 import ui
 from textInfos import DocumentWithPageTurns
 
-class FindDialog(wx.Dialog):
+
+class FindDialog(
+		gui.ContextHelpMixin,
+		wx.Dialog,  # wxPython does not seem to call base class initializer, put last in MRO
+):
 	"""A dialog used to specify text to find in a cursor manager.
 	"""
+	
+	helpId = "SearchingForText"
 
 	def __init__(self, parent, cursorManager, text, caseSensitivity, reverse=False):
 		# Translators: Title of a dialog to find text.
-		super(FindDialog, self).__init__(parent, wx.ID_ANY, _("Find"))
+		super().__init__(parent, wx.ID_ANY, _("Find"))

```

Comment:
I believe using `wx.ID_ANY` is an outdated pattern, it is the default value. Instead, just leave it out and specify the `title` keyword.
```suggestion
		super().__init__(parent, title=_("Find"))

```